verto/tests/ImageContainerTest.py

```diff
@@ -348,6 +348,81 @@ def test_image_in_numbered_list(self):
         expected_images = set()
         self.assertSetEqual(expected_images, images)
 
+    def test_image_width_value(self):
+        '''Test image rendered correctly with width value.
+        '''
+        test_string = self.read_test_file(self.processor_name, 'file_width_value.md')
+        blocks = self.to_blocks(test_string)
+        self.assertListEqual([True, False, True], [ImageContainerBlockProcessor(self.ext, self.md.parser).test(blocks, block) for block in blocks], msg='"{}"'.format(test_string))
+        html_template = self.read_test_file(self.processor_name, 'file_width_value_html_template.html', strip=True)
+        verto_extension = VertoExtension([self.processor_name], html_templates={self.tag_argument: html_template})
+        converted_test_string = markdown.markdown(test_string, extensions=[verto_extension])
+        expected_string = self.read_test_file(self.processor_name, 'file_width_value_expected.html', strip=True)
+        self.assertEqual(expected_string, converted_test_string)
+        images = self.verto_extension.required_files['images']
+        expected_images = set()
+        self.assertSetEqual(expected_images, images)
```

Comment:
please use spacing in the test methods, these big blocks are really hard to ready, see how the existing methods separate the different parts of the test